src/platform/site-wide/ebenefits/containers/EbenefitsLink.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { isLoggedIn } from 'platform/user/selectors';
```

Comment:
I think you can do `isLoggedIn as isLoggedInSelector`. Then in `mapStateToProps` -> 


```
useProxyUrl: shouldUseProxyUrl(state),
isLoggedIn: isLoggedInSelector(state),
```

Then `click` would be 

```
const click = () =>
      this.props.isLoggedIn && recordEvent({ event: 'nav-ebenefits-click' });
```

Reply:
@ncksllvn good catch!